test/e2e/integration/app-frontend/components.ts

```diff
@@ -39,6 +39,23 @@ describe('UI Components', () => {
     cy.get(appFrontend.changeOfName.deleteAttachment).should('not.exist');
   });
 
+  it('is possible to add delete confirmation for uploaded files: only proceeds on user confirmation', () => {
+    cy.interceptLayout('changename', (component) => {
+      if (component.id === 'fileUpload-changename' && component.type === 'FileUpload') {
+        component.alertOnDelete = true;
+      }
+    });
+    cy.goto('changename');
+    cy.get(appFrontend.changeOfName.upload).selectFile('test/e2e/fixtures/test.pdf', { force: true });
+    cy.get(appFrontend.changeOfName.uploadSuccess).should('exist');
+    cy.get(appFrontend.changeOfName.deleteAttachment).click();
+    cy.get(appFrontend.changeOfName.popOverCancelButton).click();
+    cy.get(appFrontend.changeOfName.uploadedTable).should('exist');
+    cy.get(appFrontend.changeOfName.deleteAttachment).click();
+    cy.get(appFrontend.changeOfName.popOverDeleteButton).click();
+    cy.get(appFrontend.changeOfName.uploadedTable).should('not.exist');
+  });
+
```

Comment:
This test and the other one is so similar, I think instead of copy-pasting it, you could probably combine both into one test. A suggestion:

```
it('should work for both upload components', () => {
  const components = [
    {
      type: 'FileUploadWithTag',
      uploader: appFrontend.changeOfName.uploadWithTag.uploadZone,
      shouldExist: appFrontend.changeOfName.uploadWithTag.editWindow,
    },
    {
      type: 'FileUpload',
      uploader: appFrontend.changeOfName.upload,
      shouldExist: appFrontend.changeOfName.uploadedTable,
    }
  ];

  cy.interceptLayout('changename', (component) => {
    for (const { type } of components) {
      // Intercept each type and set alertOnDelete
    }
  });

  for (const { uploader, shouldExist } of components) {
    // The test, with key elements differing between components
  }
});
```